spec/integration/s3_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/s3"
+require 'socket'
+require "aws-sdk"
+require "fileutils"
+require "stud/temporary"
+
+describe LogStash::Outputs::S3, :integration => true, :s3 => true do
+  let!(:minimal_settings)  {  { "access_key_id" => ENV['AWS_ACCESS_KEY_ID'],
```

Comment:
I have added a small readme for this plugin, but we need to complete it from the details of https://github.com/elasticsearch/dev/issues/471